web/src/screens/releases/ReleaseFilters.tsx

```diff
@@ -69,17 +69,22 @@ export const IndexerSelectColumnFilter = ({
 }: FilterProps<object>) => {
   const { data, isSuccess } = useQuery(ReleasesIndexersQueryOptions());
 
+  // Assign indexer name based on the filterValue (indexer.identifier)
+  const currentIndexerName = isSuccess && data 
+    ? data.find(indexer => indexer.identifier === filterValue)?.name 
+    : "Indexer";
+
   // Render a multi-select box
   return (
     <ListboxFilter
       id={id}
       key={id}
-      label={filterValue ?? "Indexer"}
+      label={currentIndexerName ?? "Indexer"}
```

Comment:
You can skip the conditional here since it's done above in `currentIndexerName`.